src/traffic-editor.md

```diff
@@ -1,3 +1,522 @@
 # Traffic Editor
 
-In this section, we shall describe the traffic-editor GUI.
+In this section, we shall describe the traffic-editor GUI and simulation tools.
+
+## Introduction and Objectives
+
+Traffic management of heterogeneous robot fleets is non-trivial. One of the
+challenges with coordinated management arises from varying semantics in
+information models used across fleets. Representations of waypoints, lanes,
+charging/docking stations, restricted zones, infrastructure  systems such as
+doors & lifts among others are subject to vendor's discretion. However,
+standardized conventions that convey the capabilities and intentions of fleets
+in a shared facility are quintessential for planning. Multi-agent participants
+in other modes of transportation such as roadways collectively adhere to a set
+of rules and conventions which minimizes chaos. More importantly, they allow for
+a new participant to readily integrate into the system by following the
+prescribed rules. Existing agents can accommodate the new participant as its
+behavior is apparent. Traffic conventions for multi-robot systems do not exist.
+The objective of the `traffic_editor` is to fill this gap by expressing the
+intentions of various fleets in a standardized, vendor neutral manner through a
+graphical interface. Collated traffic information from different fleets can then
+be exported for planning and control. A secondary objective of the
+`traffic_editor` is to facilitate generation of 3D simulations worlds which
+accurately reflect physical environments.
+
+## Overview
+
+The `traffic_editor` [repository](https://github.com/osrf/traffic_editor) is home to the `traffic_editor` GUI and tools to auto-generate simulation worlds from GUI output. 
+
+The functionality of the GUI is to load and annotate 2D floor plans with robot
+traffic along with building infrastructure information. While readily available
+3D models of sites would ease the process running simulations, in practice these
+are often unavailable or do not exist for distribution from facility owners. Floor
+plans however are relatively easier to access. Hence, the `traffic_editor` is
+designed to work with such 2D drawings to help build up accurate digital
+representations of facilities.
```

Comment:
Suggest to delete this preceding paragraph. I moved the first sentence into the previous paragraph and also tried to re-frame the problem statement about 3D/2D models. Afaik, we can convert 3D BIM files if we need to as well but currently `traffic_editor` imports only image files (which could come from 2D or 3D CAD). Totally open to changes or alternate approaches on the paragraph(s)

Reply:
Deleted https://github.com/osrf/ros2multirobotbook/pull/20/commits/7246171e3476fc0bbbb5402dae5dd21e8d163165